bin/quicksurvey

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+vanilla survey strategy
+"""
+
+import sys
+import os
+import numpy as np
+
+sys.path.insert(0, '/home/forero/desihub/desisim/py/')
+sys.path.insert(0, '/home/forero/desihub/desimodel/py/')
+sys.path.insert(0, '/home/forero/desihub/desispec/py/')
+sys.path.insert(0, '/home/forero/desihub/specter/py/')
```

Comment:
These should be configured by the user prior to running the scripts; personal directories specific to a single machine shouldn't be in the scripts themselves.